spec/rails_helper.rb

```diff
@@ -23,3 +23,9 @@ module Features
 end
 
 ActiveRecord::Migration.maintain_test_schema!
+
+RSpec::Matchers.define :appear_before do |later_content|
+  match do |earlier_content|
+    page.body.index(earlier_content) < page.body.index(later_content)
+  end
```

Comment:
Is this used anywhere?

Reply:
Woops, not anymore, that was what I was trying to user to test the sort order in the feature spec - done in sha: 9bd5721